docs/pages/identity-security/usage/alerts.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Identity Activity Center Alerts
+sidebar_label: Alerts
+description: Describes the pre-built security detections that trigger alerts in Teleport Identity Activity Center.
+tags:
+ - identity-security
+ - alerts
+ - detections
+---
+
+# Identity Activity Center Alerts
+
+The Identity Activity Center provides 50 pre-built security detections that automatically create alerts for suspicious identity-related activities across your infrastructure. These detections monitor events from Teleport and integrated services like AWS, GitHub, and Okta to identify potential security risks.
+
+## Overview
+
+Identity Activity Center alerts help you detect and respond to security threats by monitoring for:
+- Unusual authentication patterns
+- Privilege escalations
+- Configuration changes that affect security
+- Account compromises
+- Policy violations
+
+All detections are pre-configured and currently cannot be modified. The severity levels (Critical, High, Medium, Low) are defined by the Identity Security Team based on the potential impact of each detection.
+
+![Alerts](../../../img/access-graph/alerts.png)
+
+
+## Detection Categories
+
+### AWS Detections
+
+Identity Activity Center monitors your AWS infrastructure for critical security events:
+
+**Critical & High Severity:**
+- Root account activities (console login, access key creation, general activity)
+- Security settings deletions (CloudTrail, GuardDuty, flow logs)
+- EBS encryption disabled
+- DB snapshots made public
+- Exposed credential policy modifications
+- Login profile updates
+
+**Medium & Low Severity:**
+- IAM user creation
+- DB snapshot attribute modifications
+- AWS key deletion or disabling
+
+### GitHub Detections
+
+Monitor your GitHub organizations and repositories for security-relevant changes:
+
+**Critical & High Severity:**
+- Organization security updates (SAML, 2FA, OAuth restrictions)
+- Protected branch policy overrides or deletions
+- Repository access and visibility changes
+- Secret scanning alerts
+- Organization member updates
+
+**Medium & Low Severity:**
+- Repository member updates
+- Organization moderator additions
+- Integration installations
+
+**Advanced Security Detections:**
```

Comment:
The bold text is going to render in the same paragraph as the text below it. Is that the intention? If we want this to be a heading, we should start it with a `####` for semantic HTML.

Reply:
Ty. Updated this